### ISSUE TITLE
[gemini] - fetch_markets() failing - Gemini updated their html docs

### ISSUE THREAD

**Issue by drothroc** [ccxt/ccxt#7874]
Gemini html for asset information has updated requiring updating the parser.

- OS: Windows
- Programming Language version: python 3.6
- CCXT version: 1.36.94

```
gemini = ccxt.gemini()
gemini.fetch_markets()
```

```
In [6]: f.api.fetch_markets()
---------------------------------------------------------------------------
NotSupported                              Traceback (most recent call last)
<ipython-input-6-01fa698797a5> in <module>
----> 1 f.api.fetch_markets()

/usr/local/lib/python3.6/site-packages/ccxt/gemini.py in fetch_markets(self, params)
    195     def fetch_markets(self, params={}):
    196         method = self.safe_value(self.options, 'fetchMarketsMethod', 'fetch_markets_from_api')
--> 197         return getattr(self, method)(params)
    198
    199     def fetch_markets_from_web(self, params={}):

/usr/local/lib/python3.6/site-packages/ccxt/gemini.py in fetch_markets_from_web(self, params)
    221             numCells = len(cells)
    222             if numCells < 9:
--> 223                 raise NotSupported(error)
    224             #     [
    225             #         '<td>BTC',  # currency

NotSupported: gemini the Gemini API doc HTML markup has changed, breaking the parser of order limits and precision info for Gemini markets.
```

**Comment by kroitor** [comment]
I wish they'd rather add an API endpoint for limits and precision already.

**Comment by drothroc** [comment]
Completely agree and will push for it

**Comment by c0dehunter** [comment]
Is this fixed? I am still getting

```
[Not Supported] gemini the Gemini API doc HTML markup has changed, breaking the parser of order limits and
precision info for Gemini markets.
```

CCXT Version 1.37.3 (latest)
PHP 7

**Comment by kroitor** [comment]
@c0dehunter yes, it's resolved in JS/Py, looks like a minor bug in PHP is still there, i'll be back to you with a fix shortly.

**Comment by kroitor** [comment]
@c0dehunter fixed in 1.37.4+, the new build will arrive in 20 minutes. Let us know if you have further issues with it. Thx for the feedback!

**Comment by c0dehunter** [comment]
@kroitor Thank you, that was fast!
